crates/ibc/Cargo.toml

```diff
@@ -48,19 +48,19 @@ primitive-types = { version = "0.12.0", default-features = false, features = ["s
 dyn-clone = "1.0.8"
 
 [dependencies.tendermint]
-version = "=0.25.0"
+version = "0.26.0"
```

Comment:
Does tendermint-rs always bump the versions of all these component crates whenever there is a breaking change in any one of them? Just thinking if not pinning these dependencies could break builds for our customers in the future.

Reply:
Bystander, but: the key change in tendermint-rs `0.26` (and the reason we're interested in it) was to go back to "normal" use of semver, where the patch versions don't have breaking changes. So this should be fine.

Reply:
Thank you for the clarification @hdevalence. :pray: